waku/v2/node/jsonrpc/jsonrpc_callsigs.nim

```diff
@@ -1 +1,12 @@
+# Debug API
+
+proc get_waku_v2_debug_v1_info(): WakuInfo
```

Comment:
This PR doesn't remove old one, but once we do this will impact health checks and status for cluster, just FYI